### PR TITLE
#2087 New, simple limiter

### PR DESCRIPTION
**Pull request by ErwanGuichard49**

### CODE REVIEW

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo [hunk 1]
```diff
@@ -0,0 +1,39 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo:2]: Copyright

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  extends Modelica.Blocks.Interfaces.SISO;
+
+  parameter Real YMax "Upper limit of output";
+  parameter Real YMin = -YMax "Lower limit of output";
+
+equation
+  y = if u > YMax then YMax else if u < YMin then YMin else u;
+
+  annotation(
```
Comment (cossartque) [inline dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo:14]: preferred view

Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo]: Added

File: dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo (hunk 1 above)
Comment (ErwanGuichard49) [reply dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/Limiter.mo:2]: Added

**Review by cossartque — APPROVED** [review]

**Pull request merged**